pyrgg/params.py

```diff
@@ -133,6 +133,10 @@
 
 PYRGG_CONFIG_SAVE_ERROR_MESSAGE = "[Error] Failed to save config file!"
 
+PYRGG_CONFIG_LIST_MESSAGE = "Config files detected in current directory are listed below :"
+
+PYRGG_CONFIG_LOAD_MESSAGE = "Press the config index to load or any other keys to start with new one :"
```

Comment:
`Press the config index to load or any other keys to start with the new one:`

Reply:
fixed in https://github.com/sepandhaghighi/pyrgg/pull/170/commits/a48ecb418570b7cfc9e3a251396cd319a43ffe33